setup.py

```diff
@@ -157,7 +157,7 @@
     # MANIFEST.in as well.
     include_package_data=True,
     package_data={  # Optional
-        'pypahdb': ['data/*'],
+        'amespahdbpythonsuite': ['resources/*'],
```

Comment:
For pyPAHdb we had a data-directory; I like to keep things consistent between the different code-bases; either change here or in pyPAHdb---I do like calling it 'resources' better.